Broken http example: import error from poll_loop.py: StreamErrorClosed

Attempting an example for the first time, and it looks like there's a typo in the example code.

I run `componentize-py -d ../../wit -w wasi:http/proxy@0.2.0 componentize app -o http.wasm` and get this traceback:

```
Traceback (most recent call last):
  File "/home/daedalus/src/wasm-experiments/.venv/bin/componentize-py", line 8, in <module>
    sys.exit(script())
AssertionError: Traceback (most recent call last):
  File "/0/app.py", line 10, in <module>
    import poll_loop
  File "/bundled/poll_loop.py", line 17, in <module>
    from proxy.imports.streams import StreamErrorClosed, InputStream
ImportError: cannot import name 'StreamErrorClosed' from 'proxy.imports.streams' (/world/proxy/imports/streams.py). Did you mean: 'StreamError_Closed'?
```

It looks like `bundled/poll_loop.py` does indeed try to import and use `StreamErrorClosed` instead of `StreamError_Closed`, but when I try to make the change locally (so I can maybe file a PR), it doesn't appear to fix the error.

I clearly don't understand how Python imports work with wasmtime and/or componentize-py. What am I missing? Is it pulling in the import from another path or a prebuilt module somewhere?

No rush, just wanted to start learning about wasm and coming from my background with Python seemed a reasonable entry point.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "componentize-py"
-version = "0.13.2"
+version = "0.13.3"
 edition = "2021"
 exclude = ["cpython"]
 

--- a/build.rs
+++ b/build.rs
@@ -249,6 +249,8 @@ fn include(path: &Path) -> bool {
 }
 
 fn add(builder: &mut Builder<impl Write>, root: &Path, path: &Path) -> Result<()> {
+    println!("cargo:rerun-if-changed={}", path.to_str().unwrap());
+
     if path.is_dir() {
         for entry in fs::read_dir(path)? {
             add(builder, root, &entry?.path())?;

--- a/bundled/poll_loop.py
+++ b/bundled/poll_loop.py
@@ -14,7 +14,7 @@ import subprocess
 from proxy.types import Ok, Err
 from proxy.imports import types, streams, poll, outgoing_handler
 from proxy.imports.types import IncomingBody, OutgoingBody, OutgoingRequest, IncomingResponse
-from proxy.imports.streams import StreamErrorClosed, InputStream
+from proxy.imports.streams import StreamError_Closed, InputStream
 from proxy.imports.poll import Pollable
 from typing import Optional, cast
 
@@ -61,7 +61,7 @@ class Stream:
                     else:
                         return buffer
             except Err as e:
-                if isinstance(e.value, StreamErrorClosed):
+                if isinstance(e.value, StreamError_Closed):
                     if self.stream is not None:
                         self.stream.__exit__()
                         self.stream = None

--- a/examples/cli/README.md
+++ b/examples/cli/README.md
@@ -10,7 +10,7 @@ run a Python-based component targetting the [wasi-cli] `command` world.
 ## Prerequisites
 
 * `Wasmtime` 18.0.0 or later
-* `componentize-py` 0.13.2
+* `componentize-py` 0.13.3
 
 Below, we use [Rust](https://rustup.rs/)'s `cargo` to install `Wasmtime`.  If
 you don't have `cargo`, you can download and install from
@@ -18,7 +18,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v18.0.0.
 
 ```
 cargo install --version 18.0.0 wasmtime-cli
-pip install componentize-py==0.13.2
+pip install componentize-py==0.13.3
 ```
 
 ## Running the demo

--- a/examples/http/README.md
+++ b/examples/http/README.md
@@ -10,7 +10,7 @@ run a Python-based component targetting the [wasi-http] `proxy` world.
 ## Prerequisites
 
 * `Wasmtime` 18.0.0 or later
-* `componentize-py` 0.13.2
+* `componentize-py` 0.13.3
 
 Below, we use [Rust](https://rustup.rs/)'s `cargo` to install `Wasmtime`.  If
 you don't have `cargo`, you can download and install from
@@ -18,7 +18,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v18.0.0.
 
 ```
 cargo install --version 18.0.0 wasmtime-cli
-pip install componentize-py==0.13.2
+pip install componentize-py==0.13.3
 ```
 
 ## Running the demo

--- a/examples/http/app.py
+++ b/examples/http/app.py
@@ -13,7 +13,7 @@ from proxy import exports
 from proxy.types import Ok
 from proxy.imports import types
 from proxy.imports.types import (
-    MethodGet, MethodPost, Scheme, SchemeHttp, SchemeHttps, SchemeOther, IncomingRequest, ResponseOutparam,
+    Method_Get, Method_Post, Scheme, Scheme_Http, Scheme_Https, Scheme_Other, IncomingRequest, ResponseOutparam,
     OutgoingResponse, Fields, OutgoingBody, OutgoingRequest
 )
 from poll_loop import Stream, Sink, PollLoop
@@ -40,7 +40,7 @@ async def handle_async(request: IncomingRequest, response_out: ResponseOutparam)
     path = request.path_with_query()
     headers = request.headers().entries()
 
-    if isinstance(method, MethodGet) and path == "/hash-all":
+    if isinstance(method, Method_Get) and path == "/hash-all":
         # Collect one or more "url" headers, download their contents
         # concurrently, compute their SHA-256 hashes incrementally (i.e. without
         # buffering the response bodies), and stream the results back to the
@@ -61,7 +61,7 @@ async def handle_async(request: IncomingRequest, response_out: ResponseOutparam)
 
         sink.close()
 
-    elif isinstance(method, MethodPost) and path == "/echo":
+    elif isinstance(method, Method_Post) and path == "/echo":
         # Echo the request body back to the client without buffering.
 
         response = OutgoingResponse(Fields.from_list(list(filter(lambda pair: pair[0] == "content-type", headers))))
@@ -99,11 +99,11 @@ async def sha256(url: str) -> Tuple[str, str]:
 
     match url_parsed.scheme:
         case "http":
-            scheme: Scheme = SchemeHttp()
+            scheme: Scheme = Scheme_Http()
         case "https":
-            scheme = SchemeHttps()
+            scheme = Scheme_Https()
         case _:
-            scheme = SchemeOther(url_parsed.scheme)
+            scheme = Scheme_Other(url_parsed.scheme)
 
     request = OutgoingRequest(Fields.from_list([]))
     request.set_scheme(scheme)

--- a/examples/matrix-math/README.md
+++ b/examples/matrix-math/README.md
@@ -11,7 +11,7 @@ within a guest component.
 ## Prerequisites
 
 * `wasmtime` 18.0.0 or later
-* `componentize-py` 0.13.2
+* `componentize-py` 0.13.3
 * `NumPy`, built for WASI
 
 Note that we use an unofficial build of NumPy since the upstream project does
@@ -23,7 +23,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v18.0.0.
 
 ```
 cargo install --version 18.0.0 wasmtime-cli
-pip install componentize-py==0.13.2
+pip install componentize-py==0.13.3
 curl -OL https://github.com/dicej/wasi-wheels/releases/download/v0.0.1/numpy-wasi.tar.gz
 tar xf numpy-wasi.tar.gz
 ```

--- a/examples/sandbox/README.md
+++ b/examples/sandbox/README.md
@@ -8,10 +8,10 @@ sandboxed Python code snippets from within a Python app.
 ## Prerequisites
 
 * `wasmtime-py` 18.0.0 or later
-* `componentize-py` 0.13.2
+* `componentize-py` 0.13.3
 
 ```
-pip install componentize-py==0.13.2 wasmtime==18.0.2
+pip install componentize-py==0.13.3 wasmtime==18.0.2
 ```
 
 ## Running the demo

--- a/examples/tcp/README.md
+++ b/examples/tcp/README.md
@@ -11,7 +11,7 @@ making an outbound TCP request using `wasi-sockets`.
 ## Prerequisites
 
 * `Wasmtime` 18.0.0 or later
-* `componentize-py` 0.13.2
+* `componentize-py` 0.13.3
 
 Below, we use [Rust](https://rustup.rs/)'s `cargo` to install `Wasmtime`.  If
 you don't have `cargo`, you can download and install from
@@ -19,7 +19,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v18.0.0.
 
 ```
 cargo install --version 18.0.0 wasmtime-cli
-pip install componentize-py==0.13.2
+pip install componentize-py==0.13.3
 ```
 
 ## Running the demo

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -7,7 +7,7 @@ features = ["pyo3/extension-module"]
 
 [project]
 name = "componentize-py"
-version = "0.13.2"
+version = "0.13.3"
 description = "Tool to package Python applications as WebAssembly components"
 readme = "README.md"
 license = { file = "LICENSE" }

